core/tests/protractor_utils/editor.js

```diff
@@ -377,9 +386,10 @@ var addParameterChange = function(paramName, paramValue) {
 // should be specified after the ruleName as additional arguments. For example
 // with interaction 'NumericInput' and rule 'Equals' then there is a single
 // parameter which the given answer is required to equal.
-var _selectRule = function(ruleElement, interactionId, ruleName) {
+var _selectRule = function(ruleElement, interactionId, ruleName,
+  skipParameters) {
```

Comment:
@maitbayev -- sorry, this is my error. I didn't realize that _selectRule was private, so I was just surprised to see no changes to the existing tests in core/tests/protractor. This was an error on my part, though; sorry for misleading you!